mods/ra/weapons/missiles.yaml

```diff
@@ -173,193 +128,107 @@ MammothTusk:
 			Wood: 75
 			Light: 60
 			Heavy: 25
+			Concrete: 50
 		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Tank, Structure, Wall, Husk, Trees
 	Warhead@3Eff: CreateEffect
-		Explosions: med_explosion
 		ImpactSounds: kaboom12.aud
 		InvalidImpactTypes: Water, Air, AirHit
-	Warhead@4EffWater: CreateEffect
-		Explosions: small_splash
-		ImpactSounds: splash9.aud
-		ValidImpactTypes: Water
 	Warhead@5EffAir: CreateEffect
 		Explosions: med_explosion_air
 		ImpactSounds: kaboom25.aud
 		ValidImpactTypes: Air, AirHit
 
 Nike:
+	Inherits: ^AntiAirMissile
 	ReloadDelay: 15
 	Range: 7c512
 	Report: missile1.aud
-	ValidTargets: Air
 	Projectile: Missile
 		Arm: 3
-		Blockable: false
-		ContrailLength: 10
+		Inaccuracy: 0
 		Image: MISSILE
 		HorizontalRateOfTurn: 25
 		RangeLimit: 9c0
 		Speed: 341
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
 		Damage: 50
 		ValidTargets: Air
 		Versus:
 			None: 90
 			Light: 90
 			Heavy: 50
-		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
 	Warhead@3Eff: CreateEffect
 		Explosions: med_explosion_air
 		ImpactSounds: kaboom25.aud
 
 RedEye:
+	Inherits: Nike
 	ReloadDelay: 50
-	Range: 7c512
-	MinRange: 0c512
-	Report: missile1.aud
-	ValidTargets: Air
 	Projectile: Missile
-		Arm: 3
-		Blockable: false
-		ContrailLength: 10
-		Image: MISSILE
+		Inaccuracy: 0
 		HorizontalRateOfTurn: 20
-		RangeLimit: 9c0
 		Speed: 298
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
 		Damage: 40
 		ValidTargets: Air
 		Versus:
 			None: 90
 			Wood: 75
 			Light: 60
 			Heavy: 25
-		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
-	Warhead@3Eff: CreateEffect
-		Explosions: med_explosion_air
-		ImpactSounds: kaboom25.aud
 
 Stinger:
+	Inherits: ^AntiGroundMissile
 	ReloadDelay: 60
 	Range: 9c0
-	Report: missile6.aud
 	Burst: 2
 	BurstDelay: 0
-	ValidTargets: Ground, Water
 	Projectile: Missile
 		Arm: 3
-		Blockable: false
-		ContrailLength: 10
-		Image: DRAGON
+		Inaccuracy: 0
 		HorizontalRateOfTurn: 20
 		RangeLimit: 10c819
 		Speed: 170
 		CloseEnough: 149
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
 		Damage: 30
-		ValidTargets: Air, Ground, Water
 		Versus:
 			None: 30
 			Wood: 75
 			Light: 75
+			Heavy: 100
 			Concrete: 50
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Tank, Structure, Wall, Husk, Trees
 	Warhead@3Eff: CreateEffect
-		Explosions: med_explosion
-		ImpactSounds: kaboom25.aud
 		InvalidImpactTypes: Water, Air, AirHit
 	Warhead@5EffAir: CreateEffect
 		Explosions: med_explosion_air
 		ImpactSounds: kaboom25.aud
 		ValidImpactTypes: Air, AirHit
-	Warhead@4EffWater: CreateEffect
-		Explosions: med_splash
-		ImpactSounds: splash9.aud
-		ValidImpactTypes: Water
 
 StingerAA:
-	ReloadDelay: 60
-	Range: 9c0
-	Report: missile6.aud
-	Burst: 2
-	BurstDelay: 0
+	Inherits: Stinger
```

Comment:
Smudge warhead exits early if the impact happens a certain margin above ground, so I felt this was the better solution because it a) saves 1 yaml line and b) covers the (unlikely, but you never know) situation where it hits a landing aircraft right above/at ground level, where spawning a smudge is fine.